bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -213,8 +214,17 @@ public synchronized void login() {
         reLoginNeeded = false;
 
         try {
-            String urlParameters = "userId=" + urlEncode(thingConfig.getEmail()) + "&userPassword="
-                    + urlEncode(thingConfig.getPassword());
+
+            String urlParameters = "";
+
+            // if cozytouch, must use oauth server
+            if (thingConfig.getCloudPortal().equalsIgnoreCase("ha110-1.overkiz.com")) {
```

Comment:
Please use rather a constant string instead of hardcoding of "ha110-1.overkiz.com", a COZYTOUCH_PORTAL for instance?